ParcelInvoiceMaker/ParcelInvoiceMaker/Models/PhoneNumber.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PhoneNumber.swift
+//  ParcelInvoiceMaker
+//
+//  Created by JunHeeJo on 1/26/24.
+//
+
+import Foundation
+
+struct PhoneNumber {
+    let value: String
+    
+    init(_ value: String) throws {
+        let koreanPhoneNumberRegex: String = "^\\d{2,3}-?\\d{3,4}-?\\d{4}$"
+        guard  value.range(of: koreanPhoneNumberRegex, options: .regularExpression) != nil else { throw NSError() }
```

Comment:
```suggestion
        guard value.range(of: koreanNameRegex, options: .regularExpression) != nil else { throw NSError() }
```

p5;

띄어쓰기가 2개 들어간 것으로 보여요 👀

cc. [Step1](https://github.com/yagom-academy/refactoring-parcel-invoice/pull/22/files#r1468789494)

Reply:
앗... 감사합니다!